frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -80,23 +84,16 @@ private void processMacheteGeneratedEntry(MacheteGeneratedEntry macheteEntry, An
 
     val processedBranchName = branch.getText();
 
-    if (!branchNames.contains(processedBranchName)) {
-      holder
-          .newAnnotation(HighlightSeverity.ERROR,
-              getNonHtmlString("string.GitMachete.MacheteAnnotator.cannot-find-local-branch-in-repo")
-                  .format(processedBranchName))
-          .range(branch).create();
-    }
-
```

Comment:
Moving this check after the check for duplicate entries. So that `saveDocumentBeforeCheck(file)` should be completed before performing it (file content is used in updated logic).